articles/tutorials/building_2d_games/16_service_container/index.md

```diff
@@ -0,0 +1,482 @@
+
+### Game Components vs Services
+
+In MonoGame, it's important to understand the distinction between game components and services, as they serve different but complementary purposes:
+
+**Game Components** (like [**GameComponent**](xref:Microsoft.Xna.Framework.GameComponent) and [**DrawableGameComponent**](xref:xref:Microsoft.Xna.Framework.DrawableGameComponent)) are:
+
+- Modular pieces of game functionality that can be added to a game.
+- Automatically updated and/or drawn each frame through the game loop.
+- Managed by the game's component collection.
+- Typically responsible for a specific system's behavior (input handling, audio, physics, etc.).
+
+**Services**, on the other hand, are:
+
+- References to functionality that can be accessed throughout your game.
+- Registered with and retrieved from the game's services container.
+- Not automatically updated (unless they're also game components).
+- A way to share resources across different parts of your code.
+
+While these are different concepts, they often work together in practice. Many game systems are implemented as game components that also register themselves as services.
+
+### When to Use Game Components vs Services
+
+Use **Game Components** when you need:
+
+- Functionality that requires regular updates each frame.
+- Self-contained systems with their own update and/or draw logic.
+- Organization of different game subsystems.
+
+Use **Services** when you need:
+
+- Global access to a resource or system.
+- To decouple components that need to interact.
+- To share references without passing them explicitly through constructors.
+
+### When to Register a Game Component as a Service
+
+Not all game components need to be services, and not all services need to be game components. Here are some guidelines:
+
+**Register a game component as a service when:**
+
+- Multiple other components or systems need to access its functionality.
+- You want the flexibility to swap implementations later.
+- The component provides functionality that's useful throughout your game.
+
+**Keep a game component without registering it as a service when:**
+
+- It's self-contained and doesn't need to be accessed by other components.
+- It only affects a specific part of your game.
+- It doesn't provide functionality that other components need to use.
+
+## The Game Services Property
+
+At the core of MonoGame's service system is the [**Game.Services**](xref:Microsoft.Xna.Framework.Game.Services) property, which provides access to the game's [**GameServiceContainer**](xref:Microsoft.Xna.Framework.GameServiceContainer). This container has two primary methods:
+
+1. [**AddService**](xref:Microsoft.Xna.Framework.GameServiceContainer.AddService(System.Type,System.Object)): Registers a service with the container
+2. [**GetService**](xref:Microsoft.Xna.Framework.GameServiceContainer.GetService(System.Type)): Retrieves a service from the container.
+
+### Adding a Service
```

Comment:
This section, tbh, feels rushed (sorry) and feels like it is missing something, like the "anatomy of a service".

Makes me want to question:

* Why an interface
* Where does this live
* Etc

Possibly, simply, needs a real-world use case to help frame it in the readers mind.